lib/rspec/matchers.rb

```diff
@@ -646,6 +646,7 @@ def include(*expected)
     alias_matcher :a_string_including,     :include
     alias_matcher :a_hash_including,       :include
     alias_matcher :including,              :include
+    alias_matcher :have_a_member,          :include
```

Comment:
I don't want to add `have_a_member` to RSpec.  Any user who wants it can add it with a one-liner.

> Is there any other good suggestion to make it { is_expected.to include(be_an(Integer)) }` sound "good"? Maybe `it { is_expected.to have_a_member(be_an(Integer)) }`?

Sure the "noun-phrase" aliases of the matchers are specifically there to read well for situations like this.  You can use:

``` ruby
it { is_expected.to include a_kind_of Integer }
```